build.py

```diff
@@ -126,10 +126,23 @@ def unit_test():
     """
     runid = str(uuid.uuid1())
     python_version = platform.python_version()
-    utility.exec_command(
-        'pytest -l --cov mssqlcli --doctest-modules --junitxml=junit/test-{}-results.xml \
-            --cov-report=xml --cov-report=html --cov-append '
-        '-o junit_suite_name=pytest-{} '
+    utility.exec_command(('pytest -l --cov mssqlcli --doctest-modules '
```

Comment:
do you have a test run that you can show with unstable tests running separately?

Reply:
Yep, here's an [example](https://mssqltools.visualstudio.com/cli/_build/results?buildId=49027&view=results) of a test where unstable tests actually failed.